test/e2e/testsuites/dynamically_provisioned_resize_volume_tester.go

```diff
@@ -82,5 +91,43 @@ func (t *DynamicallyProvisionedResizeVolumeTest) Run(client clientset.Interface,
 		if !newSize.Equal(updatedSize) {
 			framework.Failf("newSize(%+v) is not equal to updatedSize(%+v)", newSize, updatedSize)
 		}
+
+		ginkgo.By("checking the resizing PV result")
+		newPv, _ := client.CoreV1().PersistentVolumes().Get(context.Background(), updatedPvc.Spec.VolumeName, metav1.GetOptions{})
+		newPvSize := newPv.Spec.Capacity["storage"]
+		newPvSizeStr := newPvSize.String() + "Gi"
+
+		if !(newSize.String() == newPvSizeStr) {
+			framework.Failf("newPVCSize(%+v) is not equal to newPVSize(%+v)", newSize.String(), newPvSizeStr)
+		}
+
+		ginkgo.By("checking the resizing azurefile result")
+
+		creds, err := credentials.CreateAzureCredentialFile(false)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		azureClient, err := azure.GetAzureClient(creds.Cloud, creds.SubscriptionID, creds.AADClientID, creds.TenantID, creds.AADClientSecret)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+		var resourceGroup string
+		var accountName string
+		var shareName string
+		if newPv.Spec.PersistentVolumeSource.CSI != nil {
+			resourceGroup, accountName, shareName, _, err = azurefile.GetFileShareInfo(newPv.Spec.PersistentVolumeSource.CSI.VolumeHandle)
+			framework.ExpectNoError(err, fmt.Sprintf("Error getting filesource for azurefile %v", err))
+		} else if newPv.Spec.PersistentVolumeSource.AzureFile != nil {
+			resourceGroup = creds.ResourceGroup
```

Comment:
@andyzhangx, @chewong and I think this PR broke https://testgrid.k8s.io/provider-azure-periodic#k8s-e2e-azure-file-1-19-capz `Provisioning should create a volume on demand and resize it` because the CAPZ tests don't use kubetest and the resource group env var is `AZURE_RESOURCE_GROUP`, not `RESOURCE_GROUP`.